lib/kitchen/directions/move_solutions_to_answer_key/answer_key_inner_container.rb

```diff
@@ -1,32 +1,40 @@
 # frozen_string_literal: true
 
 module Kitchen::Directions::AnswerKeyInnerContainer
-  def self.v1(chapter:, metadata_source:, append_to:, solutions_plural: true, in_appendix: false)
+  def self.v1(chapter:, metadata_source:, append_to:, options: {
+    solutions_plural: true,
+    in_appendix: false,
+    cases: false
+  })
+    options.reverse_merge!(
+      solutions_plural: true,
+      in_appendix: false,
+      cases: false
+    )
```

Comment:
Within working on this PR I needed to refactor `answer_key_inner_container` to accept options due to Rubocop complains.